addons/pylons/stringtable.xml

```diff
@@ -128,6 +143,7 @@
             <Chinese>每個派龍架需花多久時間進行整補(單位為秒)。</Chinese>
             <Chinesesimp>每个派龙架需花多久时间进行整补(单位为秒)。</Chinesesimp>
             <Korean>파일런을 재설정 하는데 걸리는 시간 (초)</Korean>
+            <German>Die benötigte Zeit um einzelne Pylonen zu ersetzen (in Sekunden)</German>
```

Comment:
`Die benötzigte Zeit, (KOMMA) um einzelne Außenlaststationen zu beladen (in Sekunden). (PUNKT)`